frontend/packages/client/src/App.sass

```diff
@@ -58,6 +58,7 @@ $border-color-light: #DBDBDB
 @import '~bulma/sass/helpers/visibility.sass'
 @import '~bulma/sass/helpers/typography.sass'
 @import '~bulma/sass/helpers/flexbox.sass'
+@import '~bulma/sass/helpers/position.sass'
```

Comment:
this helper was missing to make the blockies position:relative

<img width="238" alt="Screen Shot 2022-06-14 at 12 36 50" src="https://user-images.githubusercontent.com/7526740/173618461-f8f11c8d-b123-4a7d-a731-7097cb00b1a6.png">